content/docs/smartstreams/getting-started.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Fluvio SmartStreams
+menu: Getting Started
+weight: 10
+---
+
+SmartStreams are a feature of Fluvio that allow you to write code that filters and
```

Comment:
SmartStream are Fluvio DataStream that can be customized with users defined functions such as filter, and map